api/overdrive.py

```diff
@@ -684,9 +684,7 @@ def update_licensepool_with_book_info(self, book, license_pool, is_new_pool):
         # and if this edition is newly made, then associate pool and edition
         # as presentation_edition
         if ((not license_pool.presentation_edition) and is_new_edition): 
-            edition_changed = license_pool.set_presentation_edition(
-                policy=None
-            )
+            edition_changed = license_pool.set_presentation_edition()
```

Comment:
I'm not sure why I didn't notice this before, but recently the policy argument was removed from set_presentation_edition() and this call to that method wasn't changed.